_pages/team.html

```diff
@@ -9,132 +9,132 @@
     <h1>Our Members</h1>
     <div>
         <div>
-            <img src="/assets/img/team/erik-boesen.png">
+            <img src="/assets/img/team/members/erik-boesen.png">
             <h5>Erik Boesen</h5>
             <p>Erik is head programmer. He also works on outreach and fundraising as well as the team website and leading the development of <a href="https://github.com/frc1418/VictiScout">VictiScout</a>, our team's scouting framework. He helped create <a href="http://frcdashboard.github.io">FRC Dashboard</a>.</p>
         </div>
         <div>
-            <img src="/assets/img/team/cooper-mcguire.png">
+            <img src="/assets/img/team/members/cooper-mcguire.png">
```

Comment:
I don't think we need an extra layer of folders for the team portraits

Reply:
Okay boss, I'm on it.

Reply:
I deleted the members file folder (after moving the member portraits to the enclosing folder of course) so that's done. I'll '<img>' tags.

Reply:
I think that fixed it. (Please just merge it please).